framework_tvb/tvb/core/entities/model/model_datatype.py

```diff
@@ -229,6 +229,7 @@ class DataTypeMatrix(DataType):
     array_data_min = Column(Float)
     array_data_max = Column(Float)
     array_data_mean = Column(Float)
+    has_volume_mapping = Column(Boolean, nullable=False, default=False)
```

Comment:
How about `has_surface_mapping` ? This is another flag on the TS, quite similar to `has_volume_mapping `